spec/src/main/asciidoc/circuitbreaker.asciidoc

```diff
@@ -109,6 +109,15 @@ public void service() {
 }
 ----
 
+==== Lifecycle
+
+Unlike some other fault tolerance strategies, circuit breaker needs to store some data between invocations: the number of recent successful and failed invocations, or how long has the circuit breaker been open.
```

Comment:
Delete "Unlike some other fault tolerance strategies," as Bulkhead has the same stategy.